torchtext/experimental/datasets/raw/translation.py

```diff
@@ -120,27 +120,33 @@ def _setup_datasets(dataset_name,
                     train_filenames,
                     valid_filenames,
                     test_filenames,
-                    root='.data'):
+                    root='.data',
+                    data_select=('train', 'test', 'valid')):
     if not isinstance(train_filenames, tuple) and not isinstance(valid_filenames, tuple) \
             and not isinstance(test_filenames, tuple):
         raise ValueError("All filenames must be tuples")
 
-    src_train, tgt_train = train_filenames
-    src_eval, tgt_eval = valid_filenames
-    src_test, tgt_test = test_filenames
+    filenames = {'train': train_filenames, 'valid': valid_filenames, 'test': test_filenames}
 
+    # We stand to download a lot of files for some of
+    # these datasets. Creating a subfolder makes that more
+    # manageable.
+    download_root = root
```

Comment:
For some reason the test fails saying that 1000 lines were expected for "val", but 1014 were produced instead. The underlying raw dataset (val.de/val.en) has 1014 lines. I suspect that maybe the files we're downloading changed and the CI is using an older, cached version of them?